governance/votes/Executive vote - January 24, 2022.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Template - [Executive Vote] Temporarily prevent FLAP auctions and MKR Burn - January 24, 2022
```

Comment:
A general thing for the whole copy, which I'm sure you've considered, but want to clarify, "FLAP auction" vs. "Surplus Auction" - I know for the manual we went with Surplus Auction so may make sense to go with that? Not a bid deal either way

Reply:
I'll do both, good call.